tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Program.cs

```diff
@@ -180,6 +184,12 @@ private PipelineConvention GetPipelineConvention(string convention, PipelineGene
                 }
 
                 logger.LogInformation("Found {0} components", components.Count());
+
+                if (HasPipelineDefinitionNameDuplicates(pipelineConvention, components))
```

Comment:
This might cause our pipeline generation to start failing so you need to be careful when you enable it for the first time.

Reply:
Tested locally, it passes for all repos (up, ci, tests, weekly conventions). Minor issue with local copies of node_modules causing duplicates, but that's an edge case that I think is ok and won't affect the pipeline.